doc/api/synopsis.md

```diff
@@ -9,9 +9,43 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Firstly, make sure you have downloaded Node.js from [Node.js Official website](http://nodejs.org/#download).
+Then, follow this [installation guide](https://nodejs.org/en/download/package-manager/).
+
+
+Now, create an empty project folder called `projects`, navigate into it:
+
+Linux and Mac:
+
```

Comment:
Can you include something like this:

>We’ll be showing off a number of commands using a terminal, and those lines all start with $. You don’t need to type in the $ character; they are there to indicate the start of each command. You’ll see many tutorials and examples around the web that follow this convention: $ for commands run as a regular user, and # for commands you should be running as an administrator. Lines that don’t start with $ are typically showing the output of the previous command.

(taken from https://doc.rust-lang.org/book/second-edition/ch01-01-installation.html#installation, feel free to reword it).